app/assets/javascripts/event_staff.js

```diff
@@ -17,7 +32,47 @@ $(document).ready(function(){
     }
   });
 
+<<<<<<< HEAD
```

Comment:
Expected an identifier and instead saw '<<'.<br>Expected an operator and instead saw '<<'.<br>Expected an operator and instead saw '<'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.